### ISSUE TITLE
[Feature Request] Dark Mode Support

### ISSUE THREAD

**Issue by Xeno1177** [cengelha/NomNom#81]
A dark mode skin would be greatly appreciated.  It can help significantly with eye strain and, being visually impaired, can be an accessibility feature for some.  Clear contrast between fonts and backgrounds is also essential.  Viewing the theme in black and white is a good way to ensure good contrast.  As an example, red text on black background is invisible.

Thanks for the awesome tool!

**Comment by GrenFlem** [comment]
pleaseeee dark mode omgggg its so blinding

**Comment by GrenFlem** [comment]
its only text color change and background change

**Comment by jdm12983** [comment]
One: there is "View" then "Settings" and you can't set the theme to "Dark"; is that not the same?

If not; try VectorCMDR's NMSE [ [https://github.com/vectorcmdr/NMSE](https://github.com/vectorcmdr/NMSE) ]; like this editor; more up to date - with a "System", "Light", or "Dark" options for it/s look/theme.
